packages/overlay-sdk/src/subgraph.ts

```diff
@@ -217,4 +220,26 @@ export const getPositionDetails = async (chainId: CHAINS, account: string, marke
     console.error("Error fetching number of positions data:", error);
     return undefined;
   }
+}
+
+export const getTotalSupplyDayHistory = async (chainId: CHAINS) => {
+  invariant(chainId in CHAINS, "Unsupported chainId");
+  if (chainId === CHAINS.ArbitrumSepolia) return undefined;
```

Comment:
Why not for arbitrum sepolia? In any case, the condition could be added in the invariant so that it returns "Unsupported chainId"